.gitignore

```diff
@@ -4,11 +4,14 @@
 # linters
 .hintrc
 .stylelintrc.json
-/github
+<<<<<<< HEAD
+
+=======
+>>>>>>> 679ee18391f31aa4248300cf4475c7b07d68fa93
 
 
 # vscode
 /.vscode
 
 #test
-test.md
\ No newline at end of file
+test.md
```

Comment:
- Currently, there are some conflicts in your pull request. Kindly fix the linter errors so that your project can be reviewed properly 
  
![image](https://github.com/NalianAchei2020/Hello_Microverse/assets/89091822/3bf75266-c8bb-43f2-ba41-349e6beeaf4b)